### ISSUE TITLE
MD033 flagged for element with forward slash inside name

### ISSUE THREAD

**Issue by nschonni** [DavidAnson/markdownlint#201]
For the <ProjectTemplates/ItemTemplates> in the following block, it get's flagged by Markdownlint, but it isn't an element that Markdown renders as html
```md
**\<Visual Studio installation directory>\Common7\IDE\\<ProjectTemplates/ItemTemplates>**
```

**Comment by DavidAnson** [comment]
There are two parts of this line that look like HTML elements. The first is correctly ignored because the leading angle bracket is escaped by a backslash. The second  causes a violation to be reported because the leading angle bracket is not escaped; the double backslashes there escape themselves. However, the Markdown parser does not treat the second instance as an HTML element due to the forward slash embedded in it. The regular expression used by markdownlint doesn’t know about that exceptional case.
